src/panels/lovelace/editor/view-editor/hui-view-editor.ts

```diff
@@ -63,6 +63,25 @@ export class HuiViewEditor extends LitElement {
             },
           },
         },
+        {
+          name: "child_view",
+          selector: {
+            boolean: {},
+          },
+        },
+        ...(childView
+          ? [
+              {
+                name: "back_path",
```

Comment:
Not sure if this one should be in the UI, if there is not back path, do we just do `history.back()`?

Reply:
If a back_path is provided : go to the back path
if no back_path : history.back()

I some case you want to use the back icon as a "return to main view" instead of going on the previous page.

Reply:
The functionality is good, I just don't know if it should be in the UI, or if we should make it an advanced/yaml only function

Reply:
Agree with @piitaya, as a user in dashboards, back mostly means "go back on main view", but depending on your navigation flow `history.back` can bring you where you don't want to be.
I have this issue if using back: main view -> go to subview (#1) -> click on toolbar to another subview (#2) -> click on back -> bring me back to subview #1 instead of main view.

It happened when I was using mushroom's back chip, I fixed that by using an action chip and manually navigate to `/lovelace`.

Reply:
> The functionality is good, I just don't know if it should be in the UI, or if we should make it an advanced/yaml only function

Why would you like to keep this feature only for advanced users? It can significantly change the readability and navigation of user's dashboards.

In my opinion, the point would be "how the concept of subview can be clearly explained to users in documentation".

Reply:
I added helper on input to explain it 🙂

Reply:
I agree with Bram. We can do this automatically for 99% of users. Adding the input only confuses and makes them think they need it

Reply:
We can add it for advanced users (advanced mode).

I think we shouldn't create yaml-only configuration for new features. Advanced users are not always yaml users.
I'm an advanced user but not a yaml user 😅